mitodl/bootcamp-ecommerce
# Remove smapply and fluidreview

#### What are the relevant tickets?
Closes #477 
Closes #478 

#### What's this PR do?
Removes fluidreview and smapply code.

I kept the smapply and fluidreview fields intact in the `Profile` model so we know who came in through those apps.

#### How should this be manually tested?
Tests should pass, nothing should break, you should still be able to view what you owe for existing bootcamps and make new payments on them.
